crates/node/src/interactive_cli/call.rs

```diff
@@ -74,7 +84,7 @@ impl CallCommand {
                                         .join("\n")
                                 )
                             } else {
-                                format!("(raw): {:?}", result.cyan())
+                                format!("(raw): {:?}", String::from_utf8_lossy(&result).cyan())
```

Comment:
please revert, it's also not guaranteed the result of an execution will be a `String`